### DISCUSSION TITLE
Error in Podman Desktop Setup

### DISCUSSION THREAD

**Discussion by Rexxrally** [podman-desktop/podman-desktop#4913] [Q&A]
I installed Podman Desktop. It tells me that Podman 4.7.2 is installed but needs to be set up, and gives me a set up button.
The next screen tells me that Podman is already installed and to press the Next button to proceed, which I do.
It tells me that it couldn't find any Podman machine and to press Next to proceed to create one, which I do.
The next screen says:

This extension does not provide a component of type "createContainerProviderConnection"

I can't even find anything Googling this error message. What can I do to fix this issue?

**Comment by jeffmaury** [comment]
Which platform are you running on ?

**Reply by Rexxrally** [reply]
I'm running on Windows

**Reply by jeffmaury** [reply]
I cannot reproduce. Can you send your log file (accessible from the troubleshooting page)

**Reply by Rexxrally** [reply]
error : main ↪️ Kubeconfig path C:\Users\Dave.Sharp\.kube\config provided does not exist. Skipping.
warn : MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 extensions-started listeners added. Use emitter.setMaxListeners() to increase limit
log : main ↪️ PluginSystem: received dom-ready event from the UI
log : main ↪️ Delayed startup, flushing 7 events
log : main ↪️ System ready. Loading extensions...
log : main ↪️ Activating extension (podman-desktop.compose) with max activation time of 10 seconds
log : main ↪️ Activating extension (podman-desktop.docker) with max activation time of 10 seconds
log : main ↪️ Activating extension (podman-desktop.kind) with max activation time of 10 seconds
log : main ↪️ Activating extension (podman-desktop.kube-context) with max activation time of 10 seconds
log : main ↪️ [kube-context] starting extension kube-context
log : main ↪️ Activating extension (podman-desktop.lima) with max activation time of 10 seconds
debug : main ↪️ [lima] Could not find socket at C:\Users\Dave.Sharp\.lima\podman\sock\podman.sock
log : main ↪️ Activating extension (podman-desktop.podman) with max activation time of 10 seconds
log : main ↪️ Activating extension (podman-desktop.registries) with max activation time of 10 seconds
log : main ↪️ Activating extension (podman-desktop.docker) ended in 2196 milliseconds
log : main ↪️ Activating extension (podman-desktop.kube-context) ended in 2113 milliseconds
log : main ↪️ Activating extension (podman-desktop.lima) ended in 2106 milliseconds
log : main ↪️ Activating extension (podman-desktop.compose) ended in 2258 milliseconds
error : main ↪️ Unable to fetch the available extensions: Timeout awaiting 'lookup' for 1000ms
log : main ↪️ Activating extension (podman-desktop.registries) ended in 168 milliseconds
log : main ↪️ Fetched https://registry.podman-desktop.io/api/extensions.json in 121.7161999999953ms
debug : main ↪️ [podman] Error while pinging docker as podman Error: connect ENOENT //./pipe/docker_engine
error : main ↪️ [kind] Error: Failed to execute command: spawn kind-windows-amd64 ENOENT
log : main ↪️ Activating extension (podman-desktop.kind) ended in 2936 milliseconds
log : main ↪️ Activating extension podman-desktop.podman failed error:Error: Extension podman-desktop.podman activation timed out after 10 seconds
log : main ↪️ PluginSystem: initialization done.
log : main ↪️ Fetched https://registry.podman-desktop.io/api/extensions.json in 43.62710000001243ms

**Comment by jeffmaury** [comment]
Looks like the podman extension has not been fully initialized (timeout issue) so it has not registered the create machine capacity. Need to figure out why

**Reply by Rexxrally** [reply]
I did a complete uninstall-reinstall and it seems to have worked this time. Thanks for your help
